source/screens/caseScreens/CaseSummary.tsx

```diff
@@ -262,6 +262,7 @@ const computeCaseCardComponent = (
       buttonIconName={isClosed ? "remove-red-eye" : "arrow-forward"}
       completions={unApprovedCompletionDescriptions}
       completionsClarification={completionsClarification}
+      isRandomCheck={statusType.includes("randomCheck")}
```

Comment:
Haven't we had discussions about isRandomCheck before?

Reply:
Yea, not sure where this come from?